### ISSUE TITLE
puppet syntax pygment highlight

### ISSUE THREAD

**Issue by nanliu** [MakerPress/atlas-public-feedback#132]
@bodepd and myself got quite a few feedback that the puppet manifests pygment highlight was a bit unreadable, especially the yellow. Also the console pygment highlight was light grey and it also appeared to get strong feedback that it makes the pdf less readable. What are some ways to adjust the highlight color? It appears outside of our control in atlas.

**Comment by sandersk** [comment]
Hi @nanliu and @bodepd,

Thanks very much for the feedback on the syntax highlighting colors in Puppet Types and Providers. You are correct that the color-coding scheme is built into our toolchain and cannot currently be controlled via Atlas at this time.

As you noted, we use Pygments to do the syntax highlighting, and our color-coding scheme is a slightly modified version of Pygments' Manni theme. Our general aim is to use the same color coding across all O'Reilly titles, so that there is consistency for our readers (i.e., strings are always in orange).

That said, color coding is not especially valuable if folks are finding portions of the code unreadable. The Production team and I will look into replacing the gray used for console highlighting and the yellow used in the Puppet manifests  with colors that are more readable. We will keep you posted on our changes here.

Thanks again,
Sanders

**Comment by bodepd** [comment]
@sandersk yellow is also being used with ruby symbols.

**Comment by sandersk** [comment]
Hi @nanliu and @bodepd,

I've made some adjustments to the colors used in the PDF: the yellow is now a bit darker (closer to olive), and I've switched the gray used for console output to be 100% black. I've built a new PDF (available on the Dashboard), and if you take a look on page 6, you'll see examples of the new colors in effect.

Let me know if the revised highlighting looks OK to you.

Thanks,
Sanders
